clients/privacy-center/cypress/support/commands.ts

```diff
@@ -27,6 +27,18 @@ Cypress.Commands.add("waitUntilCookieExists", (cookieName: string, ...args) => {
   );
 });
 
+Cypress.Commands.add("waitUntilFidesInitialized", (...args) => {
+  cy.waitUntil(
+    () =>
+      cy
+        .window()
+        .its("Fides")
+        .its("initialized")
+        .then(() => true),
```

Comment:
Good example of why we need a `onInit` event 😄

Reply:
yes this will help so much!